cypress/integration/common/comments.js

```diff
@@ -1,6 +1,7 @@
 import { Given, Then, When } from "cypress-cucumber-preprocessor/steps"
 import TenureCommentsPageObjects from '../../pageObjects/tenureCommentsPage'
 import PersonCommentsPageObjects from "../../pageObjects/personCommentsPage"
+import propertyCommentsPageObjects from "../../pageObjects/propertyCommentsPage"
```

Comment:
propertyCommentsPageObjects -> PropertyCommentsPageObjects with capital P for consistency?